### PR TITLE
Worked on Upgrade, couldn't get downgrade

### PR DESCRIPTION
**Pull request by blunce24**
Downgrade isn't working - I'm not able to retrieve the customer id to delete the customer. Upgrade works well. The issue is line 32 of  app/controllers/charges_controller.rb.

### CODE REVIEW

**Comment by blunce24** [comment]
I seem to have figured it out - I added a new attribute to user, stripe_id, and populated it when I created a charge. Then I accessed it to retrieve the customer to delete when downgrading.

File: app/controllers/charges_controller.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    current_user.role = 'premium'
+    current_user.save!
+
+  rescue Stripe::CardError => e
+    flash[:error] = e.message
+    redirect_to new_charge_path
+  end
+
+  def destroy
+    cu = Stripe::Customer.retrieve(current_user.stripe_id)
```
Comment (bmneely) [inline app/controllers/charges_controller.rb]: not my favorite variable name :)

File: app/controllers/users/confirmations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
```
Comment (bmneely) [inline app/controllers/users/confirmations_controller.rb]: This class can probably be nuked

File: app/controllers/users/omniauth_callbacks_controller.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  # You should configure your model like this:
```
Comment (bmneely) [inline app/controllers/users/omniauth_callbacks_controller.rb]: Same with this one.

File: app/controllers/users/passwords_controller.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+class Users::PasswordsController < Devise::PasswordsController
```
Comment (bmneely) [inline app/controllers/users/passwords_controller.rb]: And this one

**Pull request merged**